can/parser_pyx.pyx

```diff
@@ -189,4 +190,18 @@ cdef class CANDefine():
       dv[address][sgname] = dict(zip(values, defs))
       dv[msgname][sgname] = dv[address][sgname]
 
-      self.dv = dict(dv)
+    # Signal information
```

Comment:
Not all signals are available in vals, plus you can't easily get msg name from val.